variables.sh

```diff
@@ -64,3 +64,41 @@ END
 }
 
 export -f generate_dummy_coverage_file
+
+# pick_subset
+# $1: " " seperated string
+# $2: result var
+# $3: subset num
```

Comment:
I think here (and throughout) index is a better name, then its more clear you're indexing into the list of subsets

Reply:
done